docs/DEVELOPMENT.md

```diff
@@ -0,0 +1,82 @@
+# Developer Notes
+
+This page shows how to set up basic development environment and how to test
+this project.
+
+## Getting Started
+
+### Prerequisites
+
+Make sure you have installed all of the following prerequisites on your development machine:
+
+- Git - [Download & Install Git](https://git-scm.com/downloads). OSX and Linux machines typically have this already installed.
+- Node.js - [Download & Install Node.js](https://nodejs.org/en/download/) and the npm package manager.
+
+### Installation and Running example
+
+```console
+> npm install
+> npm start
+```
+
+Now open your browser at `http://sw.localhost:3000`
+
+
+### Enabling subdomains
+
+You can enable support for subdomains with a simple nginx configuration:
```

Comment:
Maybe we can add a note that this may not be required depending on browser and system settings.

Also, it might be worth calling out that `test-e2e/reverse-proxy.js` exists and is what's used for our testing.. you can start up the reverse proxy easily with `node test-e2e/reverse-proxy.js`

Reply:
In this PR I am only moving content around, feel free to update / remove / adjust in separate PR.